### ISSUE TITLE
The problem of the newest version of OpenNMT-tf

### ISSUE THREAD

**Issue by penny9287** [OpenNMT/OpenNMT-tf#461]
I found that the newest version of OpenNMT-tf has achieved the functions that the inputters parameters and the parallel encoders parameters can be shared. But I also found the problems that if the "combine_features" is not "True" the system will  report an error. Moreover, when I train a model as usual the loss value as well as the BLEU value are not have obvious reduction in a long time. So I wonder if the code have some mistake or my configruation is wrong. The followings is my file.

params:
  optimizer: GradientDescentOptimizer
  learning_rate: 1.0
  param_init: 0.1
  clip_gradients: 5.0
  regularization:
    type: l2
    scale: 1e-4
  average_loss_in_time: true
  label_smoothing: 0.1
  beam_width: 5

train:
  bucket_width: 1
  batch_size: 2048
  batch_type: tokens
  save_checkpoints_secs: 900
  keep_checkpoint_max: 5
  save_summary_steps: 100
  train_steps: 500000
  maximum_features_length: 50
  maximum_labels_length: 50
  average_last_checkpoints: 5

eval:
  batch_size: 32
  eval_delay: 3600
  external_evaluators: BLEU

**Comment by guillaumekln** [comment]
Can you give more details:

* What was your previous version?
* What is the error?
* What model are you training?

**Comment by penny9287** [comment]
My previous version is 1.6.0 and I am training the seq2seq model.  

`source_inputter=onmt.inputters.ParallelInputter([
            onmt.inputters.WordEmbedder(
                vocabulary_file_key="source_words_vocabulary",
                embedding_size=256),
            onmt.inputters.WordEmbedder(
                vocabulary_file_key="feature_1_vocabulary",
                embedding_size=256),
            onmt.inputters.WordEmbedder(
                vocabulary_file_key="feature_2_vocabulary",
                embedding_size=256)], share_parameters=True, combine_features=True),`

if I set the combine_features=False, the system will report an error

**Comment by guillaumekln** [comment]
Why do you want to set `combine_features=False`? See https://github.com/OpenNMT/OpenNMT-tf/issues/448.

**Comment by penny9287** [comment]
> Why do you want to set `combine_features=False`? See #448.

Thanks for your reply. I think it will solve my problem.
